getting_started/install_openshift.adoc

```diff
@@ -0,0 +1,229 @@
+that are necessary in order to install {product-title} (you may have already enabled
+the first two repositories in this example):
+
+----
+$ subscription-manager repos --enable="rhel-7-server-rpms" --enable="rhel-7-server-extras-rpms" --enable="rhel-7-server-ose-3.6-rpms" --enable="rhel-7-fast-datapath-rpms"
+----
+
+[NOTE]
+====
+Instead of `rhel-7-server-ose-3.6-rpms`, use `rhel-7-server-ose-3.5-rpms` for
+OpenShift Container Platform 3.5 and `rhel-7-server-ose-3.4-rpms` for OpenShift
+Container Platform 3.4
+====
+
+This command tells your RHEL system that the tools required to install
+{product-title} will be available from these repositories. Now we need the
+{product-title} installer that is based on Ansible.
+
+[[install-package]]
+=== Install the {product-title} Package
+
+The installer for {product-title} is provided by the
+`atomic-openshift-utils` package. Install it using `yum` on both the master and
+the node, after running `yum update`:
+
+----
+$ yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion kexec-tools sos psacct
+$ yum -y update
+$ yum -y install atomic-openshift-utils
+$ yum -y install atomic-openshift-excluder atomic-openshift-docker-excluder
+$ atomic-openshift-excluder unexclude
+$ yum -y install docker
+----
+
+[[set-up-password-less-ssh]]
+=== Set up Password-less SSH Access
+
+Before running the installer on the master, set up password-less ssh access as
+this is required by the installer to gain access to the machines. On the master:
+
+----
+$ ssh-keygen
+----
+
+Follow the prompts and just hit enter when asked for passphrase. Once done:
+
+----
+for host in master.openshift.example.com node.openshift.example.com; do
+ssh-copy-id -i ~/.ssh/id_rsa.pub $host; done
+----
+
+[[run-the-installer]]
+=== Run the Installer
+
+Run the installer on the master:
+
+----
+$ atomic-openshift-installer install
+----
+
+This is an interactive install process that guides you through the various
+steps. In most cases, you want the default options. When it starts, select the
+option for {product-title}.
+Also, you are installing one master and one node and the domain name is the FQDN
+as mentioned at the start of this section (master.openshift.example.com and
+node.openshift.example.com).
+
+IMPORTANT: At the step where the installer asks you for the FQDN for the routes,
+you *must* use `apps.openshift.example.com` (or
+`cloudapps.openshift.example.com` as discussed earlier) and NOT
+`openshift.example.com`. If you make an error, you can edit the
+*_/etc/origin/master/master-config.yaml_* at the end of the install process and
+make this change yourself by looking for the `subdomain` entry.
+
+This install process takes approximately 5-10 minutes.
+
+[[start-openshift]]
+=== Start {product-title}
+
+After a successful install, use the following command to start {product-title}:
+
+----
+systemctl start atomic-openshift-master
+----
+
```

Comment:
Do we have the output for a successful install? troubleshooting for failed install?